t/55utf8_jp.t

```diff
@@ -11,6 +11,11 @@ require "lib.pl";
 
 my $dbh = DbiTestConnect($test_dsn, $test_user, $test_password, { mysql_enable_utf8 => 1, PrintError => 1, RaiseError => 1 });
 
+if (!MinimumVersion($dbh, '5.5')) {
+    plan skip_all =>
+        "SKIP TEST: You must have MySQL version 5.5 or greater for this test to run";
```

Comment:
Why is this test skipped before 5.5? On Travis it is working...
MySQL 5.1: https://travis-ci.org/perl5-dbi/DBD-mysql/jobs/227161603
MySQL 5.0: https://travis-ci.org/perl5-dbi/DBD-mysql/jobs/227161601
MySQL 4.1: https://travis-ci.org/perl5-dbi/DBD-mysql/jobs/227161599